src/index.ts

```diff
@@ -1050,11 +1051,11 @@ export class KycDao extends ApiBase {
     return chainAndAddress;
   }
 
-  private async checkValidNft(
+  private getRpcProviderForWallet(
```

Comment:
The diff looks a bit silly here, I extracted this part from `checkValidNft` so I can use it in `getValidNfts` as well and it shows that this is the changed method an the other is the new one :D